components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceComponent.java

```diff
@@ -802,6 +803,25 @@ protected void unsetClaimFilter(ClaimFilter claimFilter) {
         FrameworkServiceDataHolder.getInstance().removeClaimFilter(claimFilter);
     }
 
+    @Reference(
+            name = "approles.resolver.service",
+            service = ApplicationRolesResolver.class,
+            cardinality = ReferenceCardinality.OPTIONAL,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetAppRolesResolverService"
+    )
+    protected void setAppRolesResolverService(ApplicationRolesResolver applicationRolesResolver) {
+
+        FrameworkServiceDataHolder.getInstance().addApplicationRolesResolver(applicationRolesResolver);
+        log.debug("Application Roles Resolver is set in the Application Authentication Framework bundle.");
+
```

Comment:
remove additional new line

Reply:
Addressed in c11717ff2b929958aee0736eced5796cee627ce1